lib/reporters/base.js

```diff
@@ -31,10 +31,12 @@ var isatty = tty.isatty(1) && tty.isatty(2);
 exports = module.exports = Base;
 
 /**
- * Enable coloring by default.
+ * Enable coloring by default, except in the browser interface.
  */
 
-exports.useColors = supportsColor || (process.env.MOCHA_COLORS !== undefined);
+exports.useColors = process.env
```

Comment:
i don't think you need this check here since by your check at line 9, `supportsColor` will be undefined/null